.eslintrc.json

```diff
@@ -35,14 +35,13 @@
         "allowExpressions": true
       }
     ],
-    "@typescript-eslint/interface-name-prefix": ["error", "always"],
```

Comment:
Unfortunately, this rule was deprecated. I could not find this in a changelog, but the rule is [not displayed anymore](https://eslint.org/docs/rules/) and the compiler does not know it when running `yarn lint`.

Reply:
I never liked these interface prefixes anyway! Pfff

[maybe related?](https://github.com/typescript-eslint/typescript-eslint/issues/374)